src/components/DiaryCard/DiaryCard.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import Grid from "@mui/material/Grid";
+
+interface DiaryCardProps {
+  title: string;
+  username: string;
+  description: string;
+}
+
+const DiaryCard: React.FC<DiaryCardProps> = ({
+  title,
+  username,
+  description,
+}) => {
+  const [showMore, setShowMore] = useState(false);
+
+  const toggleShowMore = () => {
+    setShowMore(!showMore);
+  };
+
+  const cardStyle: React.CSSProperties = {
```

Comment:
don't use inline styles or separate CSS files when using MUI, use there styled utility to create custom components or use there sx prop for adding styles